src/cluster/managers/documentation/DocumentationTreeManager.ts

```diff
@@ -0,0 +1,89 @@
+import { KnownTextableChannel, User } from 'eris';
+
+import { Documentation, DocumentationManager } from './DocumentationManager';
+
+export abstract class DocumentationTreeManager extends DocumentationManager {
+    protected abstract getTree(user: User, channel: KnownTextableChannel): Awaitable<Documentation>;
+
+    async * #getFlatTree(user: User, channel: KnownTextableChannel): AsyncGenerator<Documentation> {
+        const tree = await this.getTree(user, channel);
+        yield* expandAsync([tree], d => {
+            if (d.type !== 'group')
+                return [];
+
+            return d.items.map(i => ({
+                ...i,
+                tags: [i.name, ...i.tags ?? []],
+                name: `${d.name} - ${i.name}`,
+                embed: {
+                    ...i.embed,
+                    color: i.embed.color ?? d.embed.color
+                }
+            }));
+        });
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I guess that is because you are implementing a method from the parent with a custom logic and you don't need th user and channel parameters, right?

Reply:
It was origionally going to be a method that was meant to be overridden by a derived type, but then I made it a private method and forgot to remove the params. Fixed now